runtime/src/main/java/org/corfudb/runtime/exceptions/StreamSubscriptionException.java

```diff
@@ -0,0 +1,12 @@
+package org.corfudb.runtime.exceptions;
+
+/**
+ * CorfuStore only allows one subscriber for a given namespace.
+ * If an attempt is made to register another subscriber without terminating or unsubscribing
+ * the existing subscriber on that namespace this exception will be thrown.
+ */
+public class StreamSubscriptionException extends RuntimeException {
```

Comment:
Instead of extending from RuntimeException I think we should introduce CorfuClientExceptions and extend from that. Right now our API with the application is not well-defined and that would be the right step forward.

Reply:
ok - will explore this. thanks